test/functional/unified-spec-runner/runner.ts

```diff
@@ -125,7 +157,15 @@ export async function runUnifiedTest(
     }
 
     for (const operation of test.operations) {
-      await executeOperationAndCheck(operation, entities, utilClient);
+      trace(operation.name);
+      try {
+        await executeOperationAndCheck(operation, entities, utilClient);
+      } catch (err) {
```

Comment:
Using `finally` here will let you run the `terminateOpenTransactions` helper and bubble up the error without needing to rethrow it explicitly.

Reply:
Changed to use finally.